website/docs/metrics.md

```diff
@@ -108,7 +108,7 @@ Below are the list of metrics provided by Gatekeeper:
 
     Aggregation: `LastValue`
 
-- Name: `gatekeeper_constraint_template_ingestion_count`
+- Name: `gatekeeper_constraint_template_ingestion_count_total`
```

Comment:
Thanks for the PR!

Instead of just updating the name, how about add a note at the top?
> If you are using a Prometheus client library, for counter metrics, the `_total` suffix is recommended and sometimes automatically appended by client libraries to indicate that the metric represents a cumulative total.